docs/app-bridge.md

```diff
@@ -33,9 +35,14 @@ type AppBridgeState = {
   token?: string;
   id: string;
   ready: boolean;
+  /**
+   * @deprecated - use saleorApiUrl instead
+   */
   domain: string;
   path: string;
   theme: ThemeType;
+  locale: LocaleCode; // See src/locales.ts
+  saleorApiUrl: string;
```

Comment:
@mmiszy is this exposed in AppBridgeState enough for you? Or what exactly are you trying to achieve?

At the moment `domain` is deprecated, so we have time to migrate older apps